sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -478,6 +478,7 @@ object SizeBasedWindowFunction {
 case class RowNumber() extends RowNumberLike {
   override val evaluateExpression = rowNumber
   override def sql: String = "ROW_NUMBER()"
```

Comment:
Hi, @rxin .
`sql` function is used in `SQLBuilder`.
https://github.com/apache/spark/pull/14435 will move the SQLBuilder functionality into `sql` function.